protocol/messenger_activity_center.go

```diff
@@ -48,6 +32,41 @@ func fromHexBytes(hb []types.HexBytes) [][]byte {
 	return b
 }
 
+func (m *Messenger) ActivityCenterNotifications(args ActivityCenterNotificationsArgs) (*ActivityCenterPaginationResponse, error) {
+	cursor, notifications, err := m.persistence.ActivityCenterNotifications(args.Cursor, args.Limit, args.ActivityTypes, args.ReadType, true)
+	if err != nil {
+		return nil, err
+	}
+
+	return &ActivityCenterPaginationResponse{
+		Cursor:        cursor,
+		Notifications: notifications,
+	}, nil
+}
+
+func (m *Messenger) ActivityCenterNotificationsCount(args ActivityCenterCountArgs) (*ActivityCenterCountResponse, error) {
```

Comment:
Since we're speaking in terms of Request/Response, I think the type names would be more consistent if we rename `ActivityCenterCountArgs` to `ActivityCenterCountRequest`, and `ActivityCenterNotificationsRequest` to `ActivityCenterNotificationsRequest`, and so on.

Reply:
fixed